Server/models/UserEmissions.js

```diff
@@ -0,0 +1,48 @@
+const Sequelize = require('sequelize');
+const sequelize = require('../utils/Database.js');
+
+const UserEmissions = sequelize.define('UserEmissions', {
+  id: {
+    type: Sequelize.INTEGER,
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    field: 'id' // specify the actual column name in the table
+  },
+  user_id: {
+    type: Sequelize.INTEGER,
+    allowNull: false,
+    field: 'user_id' // specify the actual column name in the table
```

Comment:
We should also include information of it being a primary key : ) 
     `  
references : {
            model : 'user',
            key : 'id'
        }`
in my file See the whole file here: